src/index.js

```diff
@@ -0,0 +1,63 @@
+import './style.css';
+import refresh from './assets/refresh.png';
+
+const description = document.querySelector('#inputid');
+const hardlist = document.createElement('div');
+const div = document.querySelector('#list');
+const todo = document.querySelector('#ToDo');
+let project;
+let index = 2;
+const completed = false;
+const pushArr = [];
+
+const list = [
+  {
+    idA: 1,
+    doneA: completed,
+    inputValA: 'wash the dishes',
+  },
+  {
+    idA: 2,
+    doneA: completed,
+    inputValA: 'complete To Do List project',
+  }, {
+    idA: index,
+    doneA: completed,
+    inputValA: description.value,
+  },
+];
```

Comment:
- [ ] Kindly use the correct task attributes keys. Each task should have **description (string)**, **index (number)**, and **completed (boolean)**.

  ```
  const list = [
    {
      index: 1,
      description: "First task",
      completed: false
    },
    {
      index: 2,
      description: "Second task",
      completed: true
    }
    ...
  ]
  ```